src/campaign/components/ItemModal/useItemForm.tsx

```diff
@@ -14,19 +14,23 @@ type Item = {
   name: string;
   description?: string;
   quantity?: number;
+  tags?: string[];
 };
 
 export type FormProps = {
   errors: Map<keyof Item, boolean>;
   // eslint-disable-next-line no-unused-vars
-  setValues: (value: { key: keyof Item; value: string | number }) => void;
+  setValues: (value: {
+    key: keyof Item;
+    value: string | number | string[];
```

Comment:
We might need to add a ticket to try and refactor this hook, starting to become quite complex with these multiple types!

Reply:
Agreed!